Swiftfin tvOS/Views/UserSignInView/UserSignInView.swift

```diff
@@ -16,6 +16,10 @@ import SwiftUI
 // TODO: change public users from list to grid
 
 struct UserSignInView: View {
+    enum FocusField: Hashable {
```

Comment:
Please put an empty line above `FocusField` as well, please.

```suggestion
    private enum FocusField: Hashable {
```